src/plugins/plot/MctPlot.vue

```diff
@@ -427,9 +427,11 @@ export default {
                 this.skipReloadOnInteraction = false;
                 this.loadMoreData(newRange, true);
             } else {
+                // If we're not panning or zooming (time conductor and plot x-axis times are not out of sync)
                 // Drop any data that is more than 1x (max-min) before min.
                 // Limit these purges to once a second.
-                if (!this.nextPurge || this.nextPurge < Date.now()) {
+                if (!this.isTimeOutOfSync
```

Comment:
I think we can declare variables for this and remove comments? something like 
```
const isPanningOrZooming = this.isTimeOutOfSync;
const isPurgeData = !isPanningOrZooming && (!this.nextPurge || this.nextPurge < Date.now()));
if (isPurgeData) {
```

Reply:
SURE!

Reply:
Done